contrib/babelfishpg_tsql/Makefile

```diff
@@ -90,6 +91,7 @@ PG_CXXFLAGS += -I$(ANTLR4_RUNTIME_INCLUDE_DIR)
 PG_CFLAGS += -g -Werror
 PG_CFLAGS += -fstack-protector-strong
 PG_CPPFLAGS += -I$(TSQLSRC) -I$(PG_SRC) -DFAULT_INJECTOR
+PG_CPPFLAGS += -DENABLE_TDS_SPATIAL
```

Comment:
Done

Reply:
in addition, we should just be aware that this functionality will be different when we are moving this change to gitfarm. since postgis will be installed by default, and we won't be able to alter the build script on a case-by-case basis.

Reply:
Yes will keep that in mind, probably we can include these flags then since PostGIS is always there so we could directly use these flags.